src/.eslintrc

```diff
@@ -25,7 +25,8 @@
     }
   },
   "rules": {
-    "@typescript-eslint/no-explicit-any": "off"
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
yeah I don't like this rule. kitbag needs it's own eslint-config

Reply:
This ^